### ISSUE TITLE
Possible system app leak

### ISSUE THREAD

**Issue by GhostlyCrowd** [M66B/XPrivacy#342]
I've been using Greenify to hibernate apps that normally run in the background. Part of this app, allows you to log what apps wake the hibernated apps or start them up.  One issue i have been having with Xprivacy is with an app called The Chive.

I have the chive fully limited from system apps, so in theory it should be oblivious to the fact that i have face book installed and should be able to start it. it should just think its not present on the device and go on its way.

However every time i start the chive it starts the Facebook app as well. and keeps it active, i have to go into Greenify and manually sleep Facebook again. Is there a way we can block launch intents and include it in the system app. This is a privacy leak as apps could be starting other apps and doing what ever they like in the background with them and their data.  

Basically what happens, the Chive sees Facebook and opens it. 

Whats expected: The Chive thinks Facebook is not installed/not present.

Take note this is not limited to just the Chive, many apps are doing this. another example is System Tuner. It keeps waking or launching Google+ with system apps limited.
